books/free-programming-books-langs.md

```diff
@@ -1127,6 +1127,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Single page apps in depth](http://singlepageappbook.com) - Mixu (HTML)
 * [Speaking JavaScript](http://speakingjs.com/es5/) - Dr. Axel Rauschmayer
 * [Standard ECMA-262 ECMAScript 2016 Language Specification](https://www.ecma-international.org/publications/standards/Ecma-262.htm) - Ecma International (HTML,PDF)
+* [Testing JavaScript Applications](https://github.com/lucasfcosta/testing-javascript-applications) - Lucas da Costa
```

Comment:
```suggestion
* [Testing JavaScript Applications](https://github.com/lucasfcosta/testing-javascript-applications) - Lucas Fernandes da Costa
```

Where is the book? Is not free at Manning... https://www.manning.com/books/testing-javascript-applications

Reply:
No, it's free on GitHub.